spaceshooter/spaceShooter.py

```diff
@@ -1,335 +1,408 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-# @Author: tasdik
-# @Contributers : Branden (Github: @bardlean86)
-# @Date:   2016-01-17
-# @Email:  prodicus@outlook.com  Github: @tasdikrahman
-# @Last Modified by:   tasdik
-# @Last Modified by:   Branden
-# @Last Modified by:   Dic3
-# @Last Modified time: 2016-10-16
-# MIT License. You can find a copy of the License @ http://prodicus.mit-license.org
-
-## Game music Attribution
+#!/usr/bin/env python #시스템 환경 변수를 파이썬으로 합니다.
+# -*- coding: utf-8 -*- #인코딩 방식은 utf-8을 이용합니다.
+# @Author: tasdik #저작자: tasdik
+# @Contributers : Branden (Github: @bardlean86) #컨트리뷰터(기여자): Branden
+# @Date:   2016-01-17 #날짜: 2016-01-17
+# @Email:  prodicus@outlook.com  Github: @tasdikrahman #이메일 주소 
+# @Last Modified by:   tasdik #최근 수정자 
+# @Last Modified by:   Branden #최근 수정자
+# @Last Modified by:   Dic3 #최근 수정자
+# @Last Modified time: 2016-10-16 ##최근 수정 시간 
+# MIT License. You can find a copy of the License @ http://prodicus.mit-license.org MIT 라이센스. 당신은 위 주소에서 라이센스 사본을 볼 수 있습니다.
+ 
+## Game music Attribution #게임음악 속성은 아래와 같습니다.
```

Comment:
Are you sure you want to add every possible language here to do not offend anyone?

In my opinion, it have to be removed.